src/hypothesis/searchstrategy/numbers.py

```diff
@@ -78,8 +78,8 @@ def distribution(random, n):
             else:
                 r &= (~sign_mask)
             return int_to_bytes(r, n)
-        byt = data.draw_bytes(size, distribution=distribution)
-        r = int_from_bytes(byt)
+        b = data.draw_bytes(size, distribution=distribution)
+        r = int_from_bytes(b)
```

Comment:
I picked this after talking w/ @DRMacIver. The reason for the short variable name is consistency w/ `r` below. If there wasn't an `r` below, I wouldn't have suggested `b`.

My initial suggestions were `bit`/`byte`.

There's really no reason to use this odd spelling.